update version block to display content (#5154)
dbt-labs/docs.getdbt.com: website/docs/docs/build/data-tests.md

Those four tests are enough to get you started. You'll quickly find you want to use a wider variety of tests—a good thing! You can also install generic data tests from a package, or write your own, to use (and reuse) across your dbt project. Check out the [guide on custom generic tests](/best-practices/writing-custom-generic-tests) for more information.

:::info

> There are generic tests defined in some open-source packages, such as [dbt-utils](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and [dbt-expectations](https://hub.getdbt.com/calogica/dbt_expectations/latest/) — skip ahead to the docs on [packages](/docs/build/packages) to learn more!

Reviewer: ```suggestion
There are generic tests defined in some open-source packages, such as [dbt-utils](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and [dbt-expectations](https://hub.getdbt.com/calogica/dbt_expectations/latest/) &mdash; skip ahead to the docs on [packages](/docs/build/packages) to learn more!
```

* Test result tables are created in a schema suffixed or named `dbt_test__audit`, by default. It is possible to change this value by setting a `schema` config. (For more details on schema naming, see [using custom schemas](/docs/build/custom-schemas).)
- A test's results will always **replace** previous failures for the same test.

## New `data_tests:` syntax

<VersionBlock lastVersion="1.7">

> The `tests` config has been renamed to `data_tests` in dbt version 1.8. To read more about this change, switch to version v1.8 in the documentation navigation menu.

Author: ```suggestion
In dbt version 1.8, we updated the `tests` configuration to `data_tests`. For detailed information on this update, please select version v1.8 from the documentation navigation menu.
```

</VersionBlock>

<VersionBlock firstVersion="1.8" lastVersion="1.8">
  

> Data tests were historically called "tests" in dbt as the only form of testing available. With the introduction of unit tests in v1.8, it was necessary to update our naming conventions and syntax. As of v1.8,  `tests:` is still supported in your YML configuration files as an alias but will be deprecated in the future in favor of `data_tests:`. 

Author: ```suggestion
Data tests were historically called "tests" in dbt as the only form of testing available. With the introduction of unit tests in v1.8, it was necessary to update our naming conventions and syntax. 

As of v1.8, `tests:` is still supported in your YML configuration files as an alias but will be deprecated in the future in favor of `data_tests:`. 
```

* Test result tables are created in a schema suffixed or named `dbt_test__audit`, by default. It is possible to change this value by setting a `schema` config. (For more details on schema naming, see [using custom schemas](/docs/build/custom-schemas).)
- A test's results will always **replace** previous failures for the same test.

## New `data_tests:` syntax

<VersionBlock lastVersion="1.7">

> In dbt version 1.8, we updated the `tests` configuration to `data_tests`. For detailed information on this update, please select version v1.8 from the documentation navigation menu.

Author: ```suggestion
In dbt version 1.8, we updated the `tests` configuration to `data_tests`. For detailed information, select version v1.8 from the documentation navigation menu.
```
